sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -111,10 +111,14 @@ public enum ServiceVersion
         /// The <see cref="ServiceVersion"/> of the service API used when
         /// making requests.
         /// </param>
+#pragma warning disable AZC0010 // TODO https://github.com/Azure/azure-sdk-for-net/issues/19575 - prepare for STG 77 beta release.
         public BlobClientOptions(ServiceVersion version = LatestVersion)
+#pragma warning restore AZC0010
         {
             if (ServiceVersion.V2019_02_02 <= version
-                && version <= LatestVersion)
+                // TODO https://github.com/Azure/azure-sdk-for-net/issues/19575 - prepare for STG 77 beta release.
+                // Change back to LatestVersion
+                && version <= ServiceVersion.V2020_08_04)
```

Comment:
I think we can define `LastVersion` and `LastStableVersion` constants in one place (or something like that, I hope the intent is clear), it will be less places to revisit when we release. And I think we'll keep repeating this procedure.

Reply:
The issue here is that the recorded tests need the actual latest version, and the live tests need the latest version - 1.

I'm not sure if we want to introduce new logic in the SDK just for testing purposes.

Reply:
Hmm then we can keep `LastVersion` as is and introduce internal constant `MaxVersion` for all these conditions. Then we make sure that `MaxVersion == LastVersion` when we release.
It's a good point that this will serve our testing. On the other hand centralizing this may reduce risk of forgetting about some package (or we need to work on solution that could remind us about this, i.e. how do we get rid of that analyzer we suppressed in this PR?) and reducing places we need to revisit.
Definitely not a blocker at this point but a convenience for us to consider.

Reply:
Fixed for everything except common.